examples/webgl_geometry_terrain_raycast.html

```diff
@@ -104,6 +104,7 @@
 				scene.add( helper );
 
 				container.addEventListener( 'mousemove', onMouseMove );
+				container.addEventListener( 'pointermove', onMouseMove );
```

Comment:
Are the `mousemove` handlers still required in this setup?

Reply:
Could be good to keep them for the best compatibility. OrbitControls and TrackballControls have kept them so far as well.

Reply:
> OrbitControls and TrackballControls have kept them so far as well.

I think you meant `touchmove`?

In this case we can remove the `mousemove` event.

Reply:
Ah true, I didn't read OrbitControls and TrackballControls carefully enough. Should maybe consider changing all examples to use pointer events instead of mouse events, and renaming "onMouseMove" functions to "onPointerMove" etc. But that's out of scope for this patch.

Reply:
I pushed an additional commit fixing this.